tests/files/packet_ubuntu20-calico-aio-ntp.yml

```diff
@@ -0,0 +1,17 @@
+---
```

Comment:
Please modify an existing test, don't add another CI job just for this because it wastes time and CI resources.

Reply:
HI @cristicalin ,

Thank you for comments very much. These problems has been fixed.